modules/sending-training-data-to-a-model.adoc

```diff
@@ -8,7 +8,7 @@ You must send training data through your model so that TrustyAI can compute base
 
 .Prerequisites
 
-* Your OpenShift cluster administrator has installed {productname-short} and enabled the TrustyAI service for the data science project where the model is deployed.
+* Your OpenShift cluster administrator has installed {productname-short} and installed the TrustyAI service for the data science project where the model is deployed.
```

Comment:
```suggestion
* Your OpenShift cluster administrator has installed {productname-short} and the TrustyAI service for the data science project where the model is deployed.
```
I don't think we need "installed" twice. I'm thinking about it this way:
"Your administrator has installed X and Y for the data science project"
vs
"Your administrator has installed X and installed Y for the data science project"
Does that make sense to you too?